Dockerfile.ocp

```diff
@@ -5,9 +5,9 @@ RUN make build
 RUN make tools
 
 RUN cp /go/src/github.com/metal3-io/baremetal-operator/config/crd/ocp/ocp_kustomization.yaml /go/src/github.com/metal3-io/baremetal-operator/config/crd/kustomization.yaml &&\
-    make manifests &&\
+    make crd &&\
```

Comment:
Do we actually need to run kustomize here at all? It looks to me like there is no difference between the checked in file  `config/crd/bases/metal3.io_baremetalhosts.yaml` and the kustomize output except for line breaks.

Reply:
config/crd/bases/metal3.io_baremetalhosts.yaml does not have the annotations :  include.release.openshift.io/self-managed-high-availability: "true"
include.release.openshift.io/single-node-developer: "true"

which are required if the CRD needs to be part of the OpenShift release. That is the only reason the kustomize is run run on the CRD here.

Reply:
I missed the part where it copied a different kustomization file above, thanks.